modules/serverless-logic-configuring-endpoint-url-openAPI-services.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-logic-configuring-endpoint-url-openAPI-services_{context}"]
+= Configuring the endpoint URL of OpenAPI services
+
+After accessing the function definitions in workflow states, you can configure the endpoint URL of OpenAPI services.
+
+.Prerequisites
+
+* You have created a {ServerlessLogicProductName} project or have access to a project with the appropriate roles and permissions to create applications and other workloads in OpenShift Container Platform.
+* You have created your {ServerlessLogicProductName} project.
+* You have access to the OpenAPI specification files.
+* You have defined the function definitions in the workflow.
+* You have the access to the defined functions in the workflow states.
+
+
+.Procedure
+
+. Locate the OpenAPI specification file you want to configure. For example, `subtraction.yaml`.
+
+. Convert the file name into a valid configuration key by replacing special characters (like `.`) with underscores and converting to lowercase. For example, `subtraction.yaml` becomes `subtraction_yaml`.
+
+. To define the configuration key, use the sanitized file name as the REST client configuration key. Set this key as an environment variable as follows:
++
+[source,sh]
+----
+quarkus.rest-client.subtraction_yaml.url=http://myserver.com
+----
+
+. To prevent hardcoding URLs in the `application.properties` file, use environment variable substitution as follows: 
```

Comment:
```suggestion
. To prevent hardcoding URLs in the `application.properties` file, use environment variable substitution, as shown in the following example: 
```